compoments/report-repair/repair-image-upload.js

```diff
@@ -5,6 +5,34 @@ import Button from '../button';
 import {serviceName} from '../../helpers/constants';
 import imageToBase64 from 'image-to-base64/browser';
 import {isMvpReleaseVersion} from '../../helpers/features';
+import { imageValidator } from '../../helpers/validators';
+
+export const ImagePreview =({image, onDelete}) => {
```

Comment:
As this component is used within multiple other component (here and [repair-description](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/43/files#diff-b631990ce338ca717f30ed135c4fa88ab9126e3d14d959fc3714ec541cf913aaR8)) , extract this component to a separate file.

As it is _generic_ (i.e. not specific to report-repair), it should be located in the components folder.

Reply:
That makes sense. All done!